python/spacewalk/satellite_tools/spacewalk-debug

```diff
@@ -268,6 +268,9 @@ if [ -d /var/lib/rhn/kickstarts ]; then
    cp -fa /var/lib/rhn/kickstarts/* $DIR/kickstarts/
 fi
 
+# Remove passwords from cobbler settings
+find $DIR/conf/cobbler -type f -name 'settings*' -exec sed -i 's/^default_password_crypted.*/default_password_crypted <replaced_by_debug_tool>/' {} \;
+
```

Comment:
I just found another file which need to replace the password.

  /etc/rhn/signing.conf

This is optional and does not always exist, It has 

```
GPGPASS=".."
```